unstructured/ingest/cli/cmds/astra.py

```diff
@@ -36,13 +36,6 @@ def get_cli_options() -> t.List[click.Option]:
                 "Note that the collection name must only include letters, "
```

Comment:
"to write into" , we can change this to connect to maybe, to indicate relationship with both use cases (src and dest)